SL-MT-TSP.py

```diff
@@ -89,19 +89,14 @@ def predhodnik(self, l, C, i):
         ...
         
     def predhodno_stanje(self, C, i):
-        # vstavi pogoj za dosegljivost nabora (če je to potrebno?)
-        CC = ()
-        for l in range(4):
-            if "indeks(i)" > C[l]: # kako lahko sem vnesem metodo iz class Ureditev?
-                CC[l] = C[l]
-            else:
-                CC[l] = "indeks(i)" # v seznamu l
-        return CC   
+        if i is None: # primer, ko ni predhodnega cilja
+            return None
+        return tuple(min(C[l], self.ureditve[l].indeks(i)) for l in range(4))
         
     def f(self, C, i):
         if (C, i) not in self.F:
-            if C == (-1, -1, -1, -1): # začetni pogoj
-                return 0
+            if C == None: # začetni pogoj

```

Comment:
Za primerjavo z `None` se običajno uporablja operator `is`:
```suggestion
            if C is None: # začetni pogoj
```